### PR TITLE
LPS-123389 Apply ModalContextProvider

### PR DESCRIPTION
**Pull request by gabrielpramos**

### CODE REVIEW

**Review by albinojunior — COMMENTED** [review]
Hey @gabrielpramos, what do you think about implement the modalContext management for the ApplyAppChangesModal and DeployAppModal?

**Comment by gabrielpramos** [comment]
It's already planned to an off story task, I think it's a good idea! :D

File: modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/table-view/EditTableViewModal.es.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+						last={
+							<>
+								<ClayButton
+									className="mr-3"
+									displayType="secondary"
+									onClick={onClose}
+									small
+								>
+									{Liferay.Language.get('cancel')}
+								</ClayButton>
```
Comment (albinojunior) [inline modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/table-view/EditTableViewModal.es.js:48]: @gabrielpramos How do you intend to call the save function from the EditTableView page on the modal footer?

Comment (gabrielpramos) [reply modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/table-view/EditTableViewModal.es.js:48]: I'll create a compound component EditTableView.Footer and export it to reutilize on modal's footer with just a few modifications to fit mockup without creating any regression bugs.
I can refine this idea with you.

**Comment by gabrielpramos** [comment]
ci:forward

**Comment by gabrielpramos** [comment]
ci:stop

**Comment by gabrielpramos** [comment]
ci:forward

**Comment by gabrielpramos** [comment]
ci:stop

**Pull request closed without merging**